tester_app/client/app/bundles/HelloWorld/startup/helloWorldAppClient.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import createStore from '../store/helloWorldStore';
+import HelloWorld from '../containers/HelloWorld';
+
+const helloWorldAppClient = props => {
```

Comment:
// Note this is what react_on_rails calls a "generator function" which is a function that takes props and returns a
// React component.
// See example: https://github.com/rackt/react-redux/blob/master/docs/api.md#example